Apps/WebClient/src/ClientApp/app/views/registrationInfo.vue

```diff
@@ -253,15 +274,19 @@ export default class RegistrationInfoComponent extends Vue {
   private signupProcessVisible: boolean = false;
   private dongleVisible: boolean = false;
   private registrationLink: string = "/registration/";
+  private regInviteOnly: boolean = false;
+  private regClosed: boolean = false;

```

Comment:
nit: reg is not too descriptive. isInviteOnly and isClosed look more readable to me